db/src/main/kotlin/edu/artic/db/models/ArticDataObject.kt

```diff
@@ -0,0 +1,17 @@
+package edu.artic.db.models
+
+data class ArticDataObject(
+        val image_server_url: String,
+        val data_api_url: String,
```

Comment:
Snake case vs camel case. I would use the moshi `@Json` annotation to clean up the names to camelcase

Reply:
will do so before merge